addons/mail/static/src/js/mail.js

```diff
@@ -538,12 +676,20 @@ openerp.mail = function (session) {
         },
 
         reinit: function() {
+            var self = this;
             var $render = $( session.web.qweb.render('mail.compose_message', {'widget': this}) );
 
             $render.insertAfter(this.$el.last());
             this.$el.remove();
             this.$el = $render;
 
+            // Initialize mail.Mention widget if current user is from employee group
+            new session.web.Model('res.users').call('has_group', ['base.group_user']).done(function(is_employee) {
```

Comment:
Quick fixes @jat-odoo, but checking group rights in reinit method not gud, it shall be called only once.